package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -513,6 +513,12 @@ class HydrogenBondLifetimes(object):
 
     def __init__(self, universe, selection1, selection2, t0, tf, dtmax,
                  nproc=1):
+        warnings.warn(
+            "This class is depracated. "
```

Comment:
The decorator also adds a note to the reST string which you will need to do manually otherwise. Something like
```reST
.. deprecated:: 2.0.0
   This analysis class will be removed in release 3.0. 
    Please use :meth:`MDAnalysis.analysis.hydrogenbonds.HydrogenBondAnalysis.autocorrelation` instead
```
or similar (making sure that the reST link works).